webpack.common.js

```diff
@@ -28,16 +28,6 @@ module.exports = {
           'css-loader',
           'sass-loader'
         ]
-      },
-      {
-        test: /\.(png|jp(e*)g|svg)$/,
```

Comment:
I assume this is for the icon we have on that page. Do you think we should exclude that specific icon instead of all the images ?

Reply:
the svg files are a part of the Icon folder. Since we are using webpack 5+, url-loader and file-loader were deprecated and replaced by asset modules. Since we are already using rollupjs, this block is not required as far as I understand.